questions/qa-bidi-unicode-controls.en.html

```diff
@@ -164,138 +178,230 @@ <h2>Changing the base direction</h2>
+<p>The following example shows how these control characters could be used in
+      plain text. It shows a tooltip in HTML that includes the title of the document linked to (which includes the abbreviation 'FAQ'), plus some text indicating the language of the destination document. Note how the text '(FAQ)' appears to the right of the Persian text. This is incorrect, since it is part of the (right-to-left) document title.</p>
     <figure class="example">
 <p><a href="qa-bidi-unicode-controls-data/eg-tooltip-wrong" onclick="showExample('qa-bidi-unicode-controls-data/eg-tooltip-wrong'); return false;"><img class="codelink" src="../icons/codelink.gif" alt="View code." title="View code for this example." /><img src="qa-bidi-unicode-controls-data/persian-link-wrong.png" alt="A tooltip without control characters." title="'SNOITSEUQ DEKSA YLTNEUQERF (faq)' [in persian]" /></a></p>
     </figure>
     <p> The correct title has the text '(FAQ)' to the left of the Persian text, as shown here.</p>
-    <figure class="example">
+
+
+<figure class="example">
 <p><a href="qa-bidi-unicode-controls-data/eg-tooltip-ok" onclick="showExample('qa-bidi-unicode-controls-data/eg-tooltip-ok'); return false;"><img class="codelink" src="../icons/codelink.gif" alt="View code." title="View code for this example." /><img src="qa-bidi-unicode-controls-data/persian-link-right.png" alt="A tooltip with control characters." title="'(faq) SNOITSEUQ DEKSA YLTNEUQERF' [in persian]" /></a></p>
-    </figure>
-    <p>To achieve the correct effect we add the two invisible control characters,  <span class="uname" style="color: green;">U+202B RIGHT-TO-LEFT EMBEDDING</span> (RLE), and <span class="uname" style="color: red;">U+202C POP DIRECTIONAL FORMATTING</span> (PDF), represented in the code snippet below as numeric character entities: </p>
-  <figure class="example">
-      <blockquote><code>
-        <bdo dir="ltr">title=&quot;'&amp;<span style="color: green;" title="RLE">#x202B;</span>...&amp;<span style="color: red;" title="PDF">#x202C;</span>' [in Persian]&quot;</bdo>
-        </code></blockquote>
-    </figure>
+</figure>
+
+
+<p>To achieve the correct effect we add the two invisible control characters,  <span class="uname" style="color: green;">U+2067 RIGHT-TO-LEFT ISOLATE</span> (RLI), and <span class="uname" style="color: red;">U+2069 POP DIRECTIONAL ISOLATE</span> (PDI), represented in the code snippet below as numeric character entities:</p>
+
+<figure class="example">
+<blockquote><code>
+<bdo dir="ltr">title=&quot;'<span style="color: green;" title="RLE">&amp;#x2067;</span>FREQUENTLY ASKED QUESTIONS (faq)<span style="color: red;" title="PDF">&amp;#x2069;</span>' [in persian]&quot;</bdo>
+</code></blockquote>
+</figure>
+
+<div id="out_tooltip" class="browser_output">
+<p class="browser_output_title">Output in your browser (mouse over for result):</p>
+<div class="browser_output_example" lang="ja">
```

Comment:
Why is this example (and other examples) tagged `lang="ja"`?

Reply:
Oops. Fixed.